### ISSUE TITLE
Need help publish wheel files to PypI (aarch64)

### ISSUE THREAD

**Issue by juntangc** [pymssql/pymssql#858]
**Describe the bug**
I have built the wheels on aarch64 (multilinux2014). But I wasn't able to upload the wheels to PyPI.

**To Reproduce**
`twine upload  dist/*` failed with the following messages
```
Uploading pymssql-2.2.8.dev61-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.8/3.8 MB • 00:00 • 41.5 MB/s
WARNING  Error during upload. Retry with the --verbose option for more details.                       
ERROR    HTTPError: 403 Forbidden from https://upload.pypi.org/legacy/                                
         The user '' isn't allowed to upload to project 'pymssql'. See                        
         https://pypi.org/help/#project-name for more information.                                    
```

**Expected behavior**
the aarch64 wheels uploaded to PyPI successfully

**Current Behavior**
the upload failed

**Comment by ski7777** [comment]
this wont work. Only the project can upload wheels to its namespace on pypi. Just wait until the next release is being published

**Comment by juntangc** [comment]
Do you know the timeline of the next release? The project has the wheels for aarch64 for the next release?

**Comment by snadampal** [comment]
Hi @ski7777 , is the next release going to have aarch64 wheel support?

**Comment by juntangc** [comment]
I have added the instructions to build the wheel for aarch64 in this PR - https://github.com/pymssql/pymssql/pull/859.

**Comment by ski7777** [comment]
No. I just prepared everything for the next release. A maintainer needs to push a tag and then the pipeline will build the wheels

**Comment by juntangc** [comment]
aarch64 also needs to go to the workflow so that the wheels can be built and tested.

**Comment by juntangc** [comment]
@ski7777 I have queued up a bunch of actions by accident.  Can you help clear the queue?  I will add another PR to use QEMU for the wheel.

**Comment by termim** [comment]
What queue you need to clear?

**Comment by juntangc** [comment]
> What queue you need to clear?

Can you clear the pending workflow actions https://github.com/pymssql/pymssql/actions? thanks,

**Comment by termim** [comment]
Sure will do. BTW you can run actions in your clone before submitting a PR.

**Comment by juntangc** [comment]
I have tested the actions in local clone.  Can you `approve and run` the workflows for PR #866?
